app/services/irs_attempts_api/encryptor.rb

```diff
@@ -0,0 +1,40 @@
+module IrsAttemptsApi
+  class Encryptor
```

Comment:
WDYT of calling this class `EnvelopeEncryptor` or `BatchEncryptor` to highlight that it's different than individual event encryption?

Reply:
👍🏼

Reply:
Updated to `EnvelopeEncryptor` in f038be97d